modules/manage/pages/security/fips-compliance.adoc

```diff
@@ -9,6 +9,11 @@ Redpanda provides FIPS-compliant cipher enforcement for brokers using OpenSSL 3.
 include::shared:partial$enterprise-license.adoc[]
 ====
 
+[IMPORTANT]
+====
+Redpanda is not fully FIPS-compliant when used with Redpanda Operator in a Kubernetes deployment.
```

Comment:
Does this also apply to the Helm chart @deniscoady? I expect so since that's what deploy RP into the base image atm.

Reply:
I think it's safe to say yes, neither the operator nor the helm chart are FIPS compliant. There are a handful of auxiliary non-FIPS containers in the helm chart that make HTTPS requests (All the jobs and init containers).